css/properties.json

```diff
@@ -5945,6 +5945,21 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "overflow-anchor": {
+    "syntax": "auto | none",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Box Model"
```

Comment:
I'm not sure CSS Box Model is right for this, is it? If "group" should map to the specification then perhaps this should be something like "CSS Scroll Anchoring"?

Reply:
Thanks. I felt it was a little bit off too. But wasn't confident enough to create a new one. 😸